src/ExpressionEvaluator/Core/Source/ExpressionCompiler/PseudoVariableUtilities.cs

```diff
@@ -54,5 +54,14 @@ internal static DkmClrCompilationResultFlags GetLocalResultFlags(this Alias alia
                     return DkmClrCompilationResultFlags.None;
             }
         }
+
+        internal static bool IsReturnValue0(this Alias alias)
```

Comment:
I struggled to find a name for this method and obviously I failed.  Its purpose is to filter out `$ReturnValue`.  What about `IsReturnValueWithoutIndex`?